### DISCUSSION TITLE
Gmail isn't rendering images from USPS Informed Delivery

### DISCUSSION THREAD

**Discussion by Robert-D** [moralmunky/Home-Assistant-Mail-And-Packages#731] [Q&A]
Hi all,
Is it just me or are others also having problems with their Gmail account not rendering images from their USPS daily digest? It looks like USPS renders the images in an <img> tag using base64 encoding, rather than attaching the images. Is manipulating something? If I download the original and open it in an outlook client it appears to show the image.

Edit: 
Looks like others are having this problem too:
[https://support.google.com/mail/thread/181914074/united-states-postal-service-usps-informed-delivery-email-images-not-rendering-correctly](https://support.google.com/mail/thread/181914074/united-states-postal-service-usps-informed-delivery-email-images-not-rendering-correctly)

**Comment by firstof9** [comment]
Works fine here.
